spec/models/help_text_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```

Comment:
Need to write the spec or delete this. Could probably test that sanitization is actually happening...